marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/WriteStepRunner.java

```diff
@@ -398,6 +400,10 @@ protected void loadStepRunnerParameters(){
             this.withStopOnFailure(Boolean.parseBoolean(stepConfig.get("stopOnFailure").toString()));
         }
 
+        if(outputURIPrefix != null && outputURIReplacement != null){
```

Comment:
I think it'd be good to change these checks to use a StringUtils method - e.g. hasText, isNotEmpty - because if one has a value and the other just has "", we're fine with that.